### PR TITLE
[pl] missing keys + stylistical fixes, better comprehension

### PR DESCRIPTION
**Pull request by makowskid**

### CODE REVIEW

**Comment by dinhquochan** [comment]
@makowskid can you check the tests? All test cases failed now because of [pl] locale.

**Comment by dinhquochan** [comment]
cc @andrey-helldar

**Comment by makowskid** [comment]
sh**, but why? I have only changed translation strings, checking

File: locales/pl/php-inline.json [hunk 1]
```diff
@@ -82,24 +82,24 @@
...
-    "not_in": "Zaznaczona wartość jest nieprawidłowa.",
+    "not_in": "Wybrana wartość jest niedozwolona.",
     "not_regex": "Format pola jest nieprawidłowy.",
     "numeric": "Pole musi być liczbą.",
     "password.letters": "To pole musi zawierać przynajmniej jedną literę.",
     "password.mixed": "To pole musi zawierać co najmniej jedną wielką i jedną małą literę.",
     "password.numbers": "To pole musi zawierać co najmniej jedną cyfrę.",
     "password.symbols": "To pole musi zawierać przynajmniej jeden symbol.",
-    "password.uncompromised": "Podane pole pojawiło się w wycieku danych. Proszę wybrać inną wartość pola.",
+    "password.uncompromised": "Pole :attribute pojawiło się w wycieku danych. Należy wybrać inną wartość.",
```
[the remark this answers is not in the document]
Comment (dinhquochan) [reply locales/pl/php-inline.json:95]: Exactly!

[the remark this answers is not in the document]
Comment (andrey-helldar) [reply locales/pl/php-inline.json:95]: I can assume it should be like this here (wrote it by analogy with other fields):
```suggestion
    "password.uncompromised": "To pole musi pojawiło się w wycieku danych. Należy wybrać inną wartość.",
```

[the remark this answers is not in the document]
Comment (makowskid) [reply locales/pl/php-inline.json:95]: pushed, hope all works now, sorry again, fat fingers

[the remark this answers is not in the document]
Comment (andrey-helldar) [reply locales/pl/php-inline.json:95]: No problem 👌

[the remark this answers is not in the document]
Comment (makowskid) [reply locales/pl/php-inline.json:95]: @andrey-helldar I think now we have a different problem
https://github.com/Laravel-Lang/lang/actions/runs/8966720071/job/24622816598#step:5:35

```
1) Tests\PluginTest::testInline
The locales/pl/json-inline.json file must not contain the value: :attribute
Podane pole :attribute pojawiło się w wycieku danych. Wybierz inną wartość pola :attribute.
Failed asserting that true is false.
```

`"The given :attribute has appeared in a data leak. Please choose a different :attribute.": "Podane pole :attribute pojawiło się w wycieku danych. Wybierz inną wartość pola :attribute."`

This test assertion is obviously wrong.

[the remark this answers is not in the document]
Comment (andrey-helldar) [reply locales/pl/php-inline.json:95]: No, the test is correct. The inline translation file must not contain a reference to the attribute name.

[the remark this answers is not in the document]
Comment (makowskid) [reply locales/pl/php-inline.json:95]: OK, you're right, this is the original `EN` version:
```
    "The given :attribute has appeared in a data leak. Please choose a different :attribute.": "The given field has appeared in a data leak. Please choose a different field."

```
I was a bit confused about the `:attribute` key here, weird.
fix pushed, again

**Pull request merged**